packages/augur-sdk/src/connector/index.ts

```diff
@@ -4,3 +4,4 @@ export * from "./seo-connector";
 export * from "./baseConnector";
 export * from "./direct-connector";
 export * from "./single-thread-connector";
+export * from "./ws-connector";
```

Comment:
I had actually removed that in a rollback that I did but I didn't break because I didn't rebuild it. So adding it back now.